src/i18n/locales/nl.json

```diff
@@ -1,37 +1,87 @@
+  "nav-recording": "Opnemen",
+  "nav-settings": "Instellingen",
+  "nav-about": "Info",
+  "review-button-discard-and-record": "Verwijderen en opnieuw opnemen",
   "save-creation-button-upload": "Uploaden naar Opencast",
   "save-creation-form-invalid": "Stel titel en presenter in",
   "save-creation-label-presenter": "Presenter",
   "save-creation-label-title": "Titel",
-  "settings-first-run": "Voordat je je scherm en/of camera kunt opnemen moet je verbinding maken met een Opencast server. Geef de volgende instellingen op.",
-  "share-desktop": "Bureaublad delen",
+  "save-creation-title": "Bijna klaar!",
+  "save-creation-title-done": "Klaar!",
+  "save-creation-subsection-title-download": "Opname downloaden",
+  "save-creation-download-button": "Downloaden",
+  "save-creation-subsection-title-upload": "Uploaden naar Opencast",
+  "save-creation-upload-to": "Upload naar <1>{{server}}</1>",
+  "save-creation-new-recording": "Start nieuwe opname",
+  "save-creation-new-recording-warning": "Wanneer u een nieuwe opname start, wordt de actuele opname in deze programma verwijderd (maar de gedownloade of geüploade opnames blijven behouden). Weet je het zeker?",
+  "confirm-discard-recordings": "Weet u zeker dat u deze opname wilt weggooien? Dit kan niet ongedaan worden gemaakt!",
+  "sources-video-display-selected": "Beeldscherm geselecteerd",
+  "sources-video-display-and-user-selected": "Beeldscherm en camera geselecteerd",
+  "sources-video-user-selected": "Camera geselecteerd",
+  "sources-video-question": "Welke videobron(nen) wilt u opnemen?",
+  "sources-video-none-available": "Uw browser/apparaat ondersteunt geen toegang tot het camerabeeld en het beeldscherm. Gebruik een ondersteunde browser.",
+  "sources-video-reselect-source": "Bron(nen) opnieuw selecteren",
+  "sources-video-no-cam-detected": "Geen camera gedetecteerd",
+  "sources-scenario-display-and-user": "Beeldscherm en camera",
+  "sources-scenario-display": "Beeldscherm",
+  "sources-scenario-user": "Camera",
+  "sources-display": "Beeldscherm",
+  "sources-user": "Camera",
+  "source-display-not-allowed-title": "Toegang tot het beeldscherm is niet mogelijk",
+  "source-display-not-allowed-text": "Probeer de bron opnieuw te selecteren. Mocht dat niet werken, laadt u deze pagina dan opnieuw of staat u deze pagina handmatig toe om toegang te krijgen tot uw beeldscherm (meestal door op een knop naast de adresbalk van uw browser te klikken).",
+  "source-user-not-allowed-title": "Toegang tot de camera is niet mogelijk",
+  "source-user-not-allowed-text": "Probeer de bron opnieuw te selecteren. Mocht dat niet werken, laadt u deze pagina dan opnieuw of staat u deze pagina handmatig toe om toegang te krijgen tot uw beeldscherm (meestal door op een knop naast de adresbalk van uw browser te klikken).
+  Probeer ook andere toepassingen die deze camera gebruiken af te sluiten: op sommige systemen kunnen camera's niet door meerdere programma' s tegelijk worden gebruikt.",
```

Comment:
Unfortunately, it's not possible to wrap strings in multiple files. This is causing a parse error.